frontend/estate-scout/src/app/property-listing/property-listing.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="property-box">
+  <div class="property-img-container">
+    <img src="{{ propertyListing.image[0].path }}" alt="primary property image" class="property-img">
+  </div>
+  <div class="property-info">
+    <p><b>{{ propertyListing.town }}</b></p>
+    <p>{{ propertyListing.price | currency:"GBP" }}</p>
+    <p *ngIf="propertyListing.propertyType == 'FOR_SALE'">For Sale</p>
+    <p *ngIf="propertyListing.propertyType == 'FOR_RENT'">For Rent</p>
```

Comment:
Let's make a method for this that returns the right text so we can keep html clean

Reply:
Created a function which formats it